tests/test_definitions/column_map_expectations/expect_column_values_to_match_regex.json

```diff
@@ -59,7 +59,7 @@
         "unexpected_index_list": [4],
         "unexpected_list": ["bee"]
       },
-      "suppress_test_for": ["sqlite", "mssql"]
+      "suppress_test_for": ["sqlite", "mssql", "bigquery"]
     },
```

Comment:
Bigquery does not allow for column names to contain spaces, which is what this test tests. Test is therefore supressed